### ISSUE TITLE
Provide option to customize log level for ChildObserver#onUncaughtException

### ISSUE THREAD

**Issue by rreimann** [reactor/reactor-netty#611]
Using Spring WebFlux behind a HAProxy we see loads of "Connection reset by peer" error messages in our logs:
```
logger_name	reactor.netty.tcp.TcpServer
message	       	[id: 0x194e0b79, L:/172.20.17.46:8013 - R:/172.20.8.32:58818] onUncaughtException(SimpleConnection{channel=[id: 0x194e0b79, L:/172.20.17.46:8013 - R:/172.20.8.32:58818]})
stack_trace	io.netty.channel.unix.Errors$NativeIoException: syscall:read(..) failed: Connection reset by peer
	          at io.netty.channel.unix.FileDescriptor.readAddress(..)(Unknown Source)
```
We identified the [HAProxy as the root cause](https://stackoverflow.com/questions/21550337/haproxy-netty-way-to-prevent-exceptions-on-connection-reset) for these error messages so there's nothing wrong with reactor-netty here. 

But since we have no chance to change the HAProxy configuration (shared infrastructure and none of the other backends has an issue with the current config) and since HAProxy is quite widespread and others might run into the same issue with the rising adoption of WebFlux I'd like to raise a feature request. It would be great if reactor-netty could provide a customization option that's accessible via WebFlux and allows for reducing the level of these messages logged at [ChildObserver#onUncaughtException](https://github.com/reactor/reactor-netty/blob/3565148829c6c80d6761a998d9b90d8b10bbae8c/src/main/java/reactor/netty/tcp/TcpServerBind.java#L222).

**Comment by violetagg** [comment]
@rreimann Is that reproducible with the latest `0.8.5.BUILD-SNAPSHOT` (we introduced this change yesterday #610 )

**Comment by rreimann** [comment]
@violetagg thx for the quick feedback - will check asap and provide feedback.

**Comment by rreimann** [comment]
👍 can confirm that the fix contained in `0.8.5.BUILD-SNAPSHOT` works as expected. The error is logged on `DEBUG` instead of `ERROR` level now:
```
2019-02-12 16:10:06.070 DEBUG [-,,,] 81353 --- [ctor-http-nio-3] reactor.netty.tcp.TcpServer              : [id: 0x2b4ead17, L:/127.0.0.1:8013 - R:/127.0.0.1:55002] onUncaughtException(SimpleConnection{channel=[id: 0x2b4ead17, L:/127.0.0.1:8013 - R:/127.0.0.1:55002]})

java.io.IOException: Connection reset by peer
        at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
        at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
```
Therefore I'm closing this issue - thanks a lot! 😃
